lib/puppet/provider/graylog_api.rb

```diff
@@ -80,7 +80,7 @@ def request(method,path,params={})
       begin
         scheme = api_tls ? 'https' : 'http'
 
-        Puppet.debug { "#{method.upcase} request for #{scheme}://#{api_server}:#{api_port}/api/#{path} with params #{params.inspect}" }
+        #Puppet.debug { "#{method.upcase} request for #{scheme}://#{api_server}:#{api_port}/api/#{path} with params #{params.inspect}" }
```

Comment:
Maybe delete these debug statements rather than commenting them if they do not help?